lib/services/messenger/datahubs/configuration.ts

```diff
@@ -1,54 +0,0 @@
-/*
```

Comment:
This git commit doesn't provide an explanation for why these files have been deleted, at a surface level it appears to be entirely unrelated to your commit message.

Reply:
Reworded commit message.